cmd/anonymizer/app/anonymizer/anonymizer.go

```diff
@@ -166,3 +197,25 @@ func filterTags(tags []model.KeyValue) []model.KeyValue {
 	}
 	return out
 }
+
+// filterCustomTags returns all tags other than allowedTags
+func filterCustomTags(tags []model.KeyValue) []model.KeyValue {
+	out := make([]model.KeyValue, 0, len(tags))
+	for _, tag := range tags {
+		if !allowedTags[tag.Key] {
+			out = append(out, tag)
+		}
+	}
+	return out
+}
+
+// hashTags converts each tag into corresponding string values
+// and then find its hash
+func hashTags(tags []model.KeyValue) []model.KeyValue {
+	out := make([]model.KeyValue, 0, len(tags))
+	for _, tag := range tags {
+		kv := model.String(tag.Key, hash(tag.AsString()))
```

Comment:
just noticed this: the keys of the tags must also be hashed

Reply:
Sure, updated the PR with the change. Thank you for catching it.